dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/SSLMutualAuthTests15.java

```diff
@@ -39,7 +40,15 @@ public String getApplication() {
 
     @After
     public void stopTestServer() throws Exception {
-        super.stopServer(true);
+        String methodName = testName.getMethodName();
+
+        @SuppressWarnings("serial") //map test-method name" to array of server error messages to ignore
+        HashMap<String, String[]> serverErrorToIgnoreRegExp = new HashMap<String, String[]>() {
+            {
+                put("testClientAuthNeedWithoutClientSideKeyStoreFor15", new String[] { "CWWKO0801E" });
+            }
+        };
+        super.stopServer(true, serverErrorToIgnoreRegExp.get(methodName));
```

Comment:
Why create and use a new map each call?  The use of map seems more obscure to understand than something like this:

```
if ("testClientAuthNeedWithoutClientSideKeyStoreFor15".equals(methodName)) {
    super.stopServer(true, "CWWKO0801E");
} else {
    super.stopServer(true);
}
```